Document `--cache` flag. (#9427)
vercel/turborepo: docs/repo-docs/reference/run.mdx

  changed.
</Callout>

### `--cache <options>`

Default: `local:rw,remote:rw`

Specify caching sources for the run. Accepts a comma-separated list of options:

- `local`: Use the local filesystem cache
- `remote`: Use the Remote Cache

> When an option is omitted, it is neither read nor written to.

Reviewer: smh don't end on a preposition
```suggestion
When an option is omitted, reading and writing are both disabled
```

Each option must be followed by one of:

- `rw`: Read and write
- `r`: Read only
- `w`: Write only

```bash title="Terminal"
# Read and write to local cache, read only to Remote Cache
turbo run build --cache=local:rw,remote:r

# Read-only to local cache, read write to Remote Cache
turbo run build --cache=local:r,remote:rw

> # Read and write to local cache, no Remote Cache activity

Author: I wasn't sure if this was correct. Can someone confirm?

Author: We talked about this to make sure and it is correct!
